components/suggestion/suggestion_date.tsx

```diff
@@ -5,7 +5,7 @@ import React from 'react';
 
 import Popover from 'components/widgets/popover';
 
-type SuggestionItem = {
+export type SuggestionItem = {
```

Comment:
Does this still need to be exported? I can't see it being used anywhere else in this PR